### PR TITLE
Ledger: WebHid transport if supported, use BIP44 address standard

### PR DESCRIPTION
**Pull request by digital-flowers**
### What does this PR do?

- Use WebHid transport if available and U2F as a fallback, recently the latest chrome  removed the U2F which lead to the ledger subprovider used in many libraries to stop working, see: https://twitter.com/MetaMask/status/1397659854446022656
- Use BIP44 Address standard, see: https://github.com/ethereum/EIPs/issues/84

Note you need to accept hid device from chrome UI: 

Screenshot 2021-06-02 at 6 12 38 PM

You can already test the ledger connector on our website https://mimo.capital/  :)

### CODE REVIEW

**Comment by Dire-0x** [comment]
Hey, what is the status of this?

File: packages/ledger-connector/src/index.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import { ConnectorUpdate } from '@web3-react/types'
 import { AbstractConnector } from '@web3-react/abstract-connector'
 import Web3ProviderEngine from 'web3-provider-engine'
-import { ledgerEthereumBrowserClientFactoryAsync } from '@0x/subproviders/lib/src' // https://github.com/0xProject/0x-monorepo/issues/1400
-import { LedgerSubprovider } from '@0x/subproviders/lib/src/subproviders/ledger' // https://github.com/0xProject/0x-monorepo/issues/1400
 import CacheSubprovider from 'web3-provider-engine/subproviders/cache.js'
 import { RPCSubprovider } from '@0x/subproviders/lib/src/subproviders/rpc_subprovider' // https://github.com/0xProject/0x-monorepo/issues/1400
+import { LedgerSubprovider } from 'subprovider'
```
Comment (Dire-0x) [inline packages/ledger-connector/src/index.ts:6]:
```suggestion
import { LedgerSubprovider } from './subprovider'
```

Did you mean this?

Comment (digital-flowers) [reply packages/ledger-connector/src/index.ts:6]: with the current eslint config both works

Comment (digital-flowers) [reply packages/ledger-connector/src/index.ts]: thanks

**Comment by ju1cec** [comment]
Hi. this PR worked great to add support for Chrome 91. thanks!

**Comment by digital-flowers** [comment]
Is this repo dead?

**Comment by EdoAPP** [comment]
It'd be nice to get this merged. Does it need an extra review?

**Comment by NoahZinsmeister** [comment]
thanks for the work on this, and sorry everyone for the very delayed response. going to be focusing efforts on the (beta) v8 version of web3-react going forward, and atm there are no plans for native ledger support (preference is instead to use hardware wallets through metamask or frame), so i'm going to close this

**Pull request closed without merging**